cmd/deploy/deploy.go

```diff
@@ -439,7 +439,7 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 				}
 			}
 			if !utils.LoadBoolean(model.OktetoWithinDeployCommandContextEnvVar) {
-				if err := dc.showEndpoints(ctx, &EndpointsOptions{Name: deployOptions.Name}); err != nil {
+				if err := dc.showEndpoints(ctx, &EndpointsOptions{Name: deployOptions.Name, Namespace: deployOptions.Manifest.Namespace}); err != nil {
```

Comment:
just to double check, if the namespace is not specified in the manifest, is it being initialized with the context one or the one specified in the command, right?

Reply:
yes, the value in the manifest is the one taking into account arguments and manifest values